index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>My First Page</title>
+</head>
+<body>
+    <h1>My First Page</h1>
+    <p><a href="/about/index.html"></a></p>   
```

Comment:
**Very important.** Check your href attribute. If you want to use a file inside a folder, you must start your relative path with "./"
Also link should have text or image. otherwise, it will be non-clickable/invisible.